examples/meta/src/multiclass/multiclass_machine.sg.in

```diff
@@ -0,0 +1,34 @@
+CSVFile f_feats_train("@SHOGUN_DATA@/classifier_4class_2d_linear_features_train.dat")
+CSVFile f_feats_test("@SHOGUN_DATA@/classifier_4class_2d_linear_features_test.dat")
+CSVFile f_labels_train("@SHOGUN_DATA@/classifier_4class_2d_linear_labels_train.dat")
+
+#![create_features]
+Features features_train = features(f_feats_train)
+Features features_test = features(f_feats_test)
+Labels labels_train = labels(f_labels_train)
+#![create_features]
+
+#![create_kernel]
+Kernel kernel_gaussian = kernel("GaussianKernel", log_width=2.1)
+#![create_kernel]
+
+#![choose_strategy]
+MulticlassStrategy one_versus_rest=multiclass_strategy("MulticlassOneVsRestStrategy")
+#![choose_strategy]
+
+#![create_classifier]
+Machine classifier = machine("LibSVM", epsilon=0.00001)
+#![create_classifier]
+
+#![create_machine]
+Machine multiclass_machine =  machine("KernelMulticlassMachine", multiclass_strategy=one_versus_rest, kernel=kernel_gaussian, machine= classifier, labels=labels_train)
+#![create_machine]
+
+#![train_and_apply]
+multiclass_machine.train(features_train)
+Labels labels_predict = multiclass_machine.apply(features_test)
+#![train_and_apply]
+
+#![extract_labels]
+RealVector output = labels_predict.get_real_vector("labels")
```

Comment:
Maybe also get an accuracy evaluation? Just to see how the model performs.

Reply:
I wouldnt actually to keep these short.

Reply:
OK!